numalogic/models/threshold/_median.py

```diff
@@ -0,0 +1,87 @@
+from collections.abc import Sequence
+from typing import Optional
+
+import numpy as np
+import numpy.typing as npt
+from typing_extensions import Self, Final
+
+from numalogic.base import BaseThresholdModel
+from numalogic.tools.exceptions import InvalidDataShapeError, ModelInitializationError
+
+_INLIER: Final[int] = 0
+_OUTLIER: Final[int] = 1
```

Comment:
I think we should add this to config too. Like we have for min_threshold

Reply:
Not really. It is used in the `predict` method for which the output is just Boolean. You would never want anything else here.